workflows/raredisease.nf

```diff
@@ -181,6 +182,22 @@ workflow RAREDISEASE {
     )
     ch_versions = ch_versions.mix(CALL_STRUCTURAL_VARIANTS.out.versions)
 
+    // STEP 2.1: GENS
+    if (params.gens_switch) {
+        GENS (
+            ch_mapped.bam_bai,
+            CALL_SNV.out.vcf,
+            ch_references.genome_fasta,
+            ch_references.genome_fai,
+            file(params.gens_interval_list),
```

Comment:
This needs to be defined in nextflow_schema.json, and mapped in [main.nf](https://github.com/nf-core/raredisease/blob/dev/main.nf).